addons/project/tests/test_project_flow.py

```diff
@@ -276,3 +276,20 @@ def test_rating(self):
         self.assertEqual(rating_good.parent_res_id, self.project_goats.id)
         self.assertEqual(self.project_goats.rating_percentage_satisfaction, 50)
         self.assertEqual(self.project_pigs.rating_percentage_satisfaction, -1)
+
+    def test_task_with_no_project(self):
+        task_without_project = self.env['project.task'].with_context({'mail_create_nolog': True}).create({
```

Comment:
Perhaps add a comment to explain the goal of this test since it is not really a standard test :)

Reply:
Done :) See : https://github.com/odoo/odoo/pull/70576/files